spec/requests/zips_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Download a zip file of all attached files', type: :request do
```

Comment:
Do you think we should add a test case for the no downloadable files branch?

Reply:
I've added such a test.

Reply:
Thank you